src/sign-transaction/SignTransaction.js

```diff
@@ -0,0 +1,33 @@
+class Api {
+    /** @param {Nimiq.Transaction} transaction */
```

Comment:
The request is given a plain JSON object containing transaction data, not a `Nimiq.Transaction` object.